services/QuillLMS/spec/controllers/statuses_controller_spec.rb

```diff
@@ -53,18 +53,35 @@
   end
 
   describe "#sidekiq_queue_latency" do
-    let(:queue_latency_pairs) do
-      {
+    it 'should build a hash of queue.name => queue.latency for each queue' do
+      queue_latency_pairs = {
         critical: 1.14,
         critical_external: 0.34,
         default: 1.44,
         low: 0,
         migration: 0
-
       }
+      queue_doubles = queue_latency_pairs.map do |name, latency|
+        d = double
+        allow(d).to receive(:name).and_return(name)
+        allow(d).to receive(:latency).and_return(latency)
+        d
+      end
+
+      allow(Sidekiq::Queue).to receive(:all).and_return(queue_doubles)
+
+      expect {
```

Comment:
I don't think you gain anything from this expects. If there is an error in the method, that would inherently fail the test.